docs/usage/persistent-volumes.md

```diff
@@ -0,0 +1,27 @@
+# Persistent volumes
+
+Minishift supports [PersistentVolumes](http://kubernetes.io/docs/user-guide/persistent-volumes/) of type `hostPath`.
```

Comment:
What about on line 3 and the tmpfs on line 6...anyways like you point out, a lot of things might change.